packages/playwright/tests/viewports.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { test } from '../src';
+
+test('snapshots capture the correct viewport size', async ({ page }) => {
+  await page.goto('/viewports');
+});
+
+test.describe('hardcoded viewport', () => {
+  test.use({ viewport: { width: 800, height: 720 } });
+
+  test('snapshots capture the correct viewport size', async ({ page }) => {
+    await page.goto('/viewports');
+  });
+});
```

Comment:
Looks like some of the tests from the Cypress viewports aren't present here.

Reply:
@skitterm the one that's not present here is a test showing that `cy.viewports()` doesn't work, so isn't relevant here.